templates/_layouts/docs.html

```diff
@@ -88,7 +88,7 @@
               {{ side_nav_item("/docs/patterns/pull-quote", "Quotes") }}
               {{ side_nav_item("/docs/patterns/search-and-filter", "Search and filter") }}
               {{ side_nav_item("/docs/patterns/search-box", "Search box") }}
-              {{ side_nav_item("/docs/patterns/segmented-control", "Segmented control", 'new', 'positive') }}
+              {{ side_nav_item("/design/patterns/segmented-control", "Segmented control", 'new', 'positive') }}
```

Comment:
Without changing this the docs automatically land on "implementation" which is the second tab so it looks strange. Should this be a step we add to the guide when someone adds design guidelines?